Attribute/AsEntityListener.php

```diff
@@ -12,6 +12,7 @@ class AsEntityListener
 {
     public function __construct(
         public ?string $event = null,
+        public ?int $priority = null,
```

Comment:
needs to be moved as a last parameter, otherwise it's a BC break

Reply:
Thank you. Moved as last parameter in constructor.